db/seeds/01_users_seeds.sql

```diff
@@ -1,30 +1,30 @@
-insert into users (id, name, email, phone_number, password) values (1, 'Ethelyn Scamwell', 'escamwell0@uiuc.edu', '531-607-6170', 'jB7#&nm1*MAuJ&N');
```

Comment:
I kept having troubles when testing favorite/unfavorite routes, so I reached out for help and below were the answers.
"we defined ids serial so postgres automatically gives IDs and counts them. 
If we give ids manually when seeding, it erros out when we try to create an entry."

so I removed ids from seeds.